client/content/pages/sk/about.md

```diff
@@ -0,0 +1,3 @@
+The Satoshi Nakamoto Institute was founded in November 2013 to advance and preserve knowledge of Bitcoin’s history, economics, and technology.
```

Comment:
Please remove any pages in `content/pages/sk/` that are still in English. i18next already defaults to the English version.